build.gradle.kts

```diff
@@ -77,7 +77,8 @@ subprojects {
   pluginManager.withPlugin("java") {
     // javaReleaseVersion can be set to override the global version
     val jvmTargetProvider =
-      provider<String> { findProperty("moshix.javaReleaseVersion") as? String? }
+      providers
+        .gradleProperty("moshix.javaReleaseVersion")
```

Comment:
The problem is here unfortunately, `providers.gradleProperty()` doesn't work for subprojects, only root projects